update rn for CommonTypes 3.2.40 (#20107)
demisto/content: Packs/CommonTypes/ReleaseNotes/3_2_40.md

- **Groups** - changed the type from *longText* to *markdown*
- **Short Description** - changed the type from *longText* to *markdown*
- **Applications** - changed the type from *longText* to *markdown*
- **Note:** 
  - In order for the changes to take effect, server versions < 6.8 should re-install the *CommonTypes* pack. If any warnings appear, click the *Install anyway* button.

>   - Re-installing the pack would reset existing configurations for those types configured. For example, incident type configured for a certain playbook would no longer be configured to use.

Reviewer: ```suggestion
  - Re-installing the pack would reset existing configurations for those types configured. For example, an incident type configured for a certain playbook would no longer be configured.
```
